pinot-core/src/main/java/org/apache/pinot/core/segment/creator/impl/inv/text/LuceneTextIndexCreator.java

```diff
@@ -50,6 +58,15 @@
   private final Directory _indexDirectory;
   private final IndexWriter _indexWriter;
 
+  public static final CharArraySet ENGLISH_STOP_WORDS_SET =
+      new CharArraySet(Arrays.asList(
```

Comment:
Dont we have this in Lucene StopAnalyzer?
StopAnalyzer.ENGLISH_STOP_WORDS_SET

Reply:
No this is not in StopAnalyzer. That also takes a passed list of stopwords.

The list is borrowed from EnglishAnalyzer. The reason for not directly using EnglishAnalyzer is because it does word stemming and reduces words to their root form. I don't think we need that.

Reply:
StopAnalyzer can't be used either because it tokenizes input text at each character. So we stick to StandardAnalyzer which uses StandardTokenizer (based on unicode general purpose text segmentation algorithm)

Reply:
Thanks add this comment in the code.